src/cmd-buildextend-metal

```diff
@@ -274,10 +274,16 @@ platforms_json="${tmp_builddir}/platforms.json"
 yaml2json "${configdir}/platforms.yaml" "${platforms_json}"
 
 # Currently we only support OSBuild for qemu and metal disk images
+osbuild_extra_args=()
+osbuild_mpp_config="coreos.osbuild.${basearch}.mpp.yaml"
 if [ "${image_type}" == "qemu" ] || [ "${image_type}" == "metal" ] || [ "${image_type}" == "metal4k" ]; then
     # Right now we don't have support fully fleshed out for secex on s390x
     if [[ $secure_execution -ne "1" ]]; then
         OSBUILD_SUPPORTED=1
+    elif [[ -f "${hostkey}" ]]; then
```

Comment:
On other parts we currently use
```
if [ -z "${hostkey}" ]; then
```
It should be kept the same here so
```
if [ -n "${hostkey}" ]; then
```

Reply:
Here `-f` is used by intention - for local build it makes sense to ensure, that `hostkey` indeed exists

Reply:
Makes sense to me